pkg/scheduler/framework/runtime/framework.go

```diff
@@ -737,10 +737,15 @@ func (f *frameworkImpl) RunPreFilterPlugins(ctx context.Context, state *framewor
 			}
 
 			// When PreFilterResult filters out Nodes, the framework considers Nodes that are filtered out as getting "UnschedulableAndUnresolvable".
-			return result, framework.NewStatus(framework.UnschedulableAndUnresolvable, msg)
+			returnStatus = framework.NewStatus(framework.UnschedulableAndUnresolvable, msg)
+			break
 		}
 	}
-	return result, returnStatus
+	var unscheduledPlugins sets.Set[string]
```

Comment:
Rethink about it, can we just change `pluginsWithNodes` to `sets.Set[string]`? Any downside if we do so?

Reply:
yes. change pluginsWithNodes to sets.Set[string], we can return pluginsWithNodes instead of `unscheduledPlugins `.